src/main/java/com/devcycle/sdk/server/common/interceptor/CustomHeaderInterceptor.java

```diff
@@ -23,7 +23,7 @@ public CustomHeaderInterceptor(IRestOptions restOptions) {
     public Response intercept(Chain chain) throws IOException {
         Request request = chain.request();
 
-        if (restOptions != null) {
+        if (restOptions != null && chain.request().url().url().getHost().contains("devcycle")) {
```

Comment:
what if we move the SSE connection to our domain?

Reply:
This was from testing, need to remove